configs/cts1/auxiliary_software_files/packages.yaml

```diff
@@ -23,6 +32,11 @@ packages:
     - spec: intel-oneapi-mkl@2022.1.0
```

Comment:
@rfhaque why do we need oneapi here?

Reply:
> @rfhaque why do we need oneapi here?

@pearce intel-oneapi-mkl is the blas and lapack implementation on cts1 machine